### ISSUE TITLE
.net DataLayer client UnsusbscribeAll() bug

### ISSUE THREAD

**Issue by ArcherThompson** [boschrexroth/ctrlx-automation-sdk#31]
When preforming an UnsubscribeAll no further subscriptions can be made without disposing of the subscription and recreating it. using .Unsubscribe(yourAddress) works fine. This is repeatable and the object is never shown as disposed

**Comment by ZmRZjFuDsDiR4qi5801F** [comment]
You're right, calling UnsubscribeAll() invalidates and releases the subscription. We improved the API docs for this methods regarding:
Thank you for reporting this using the ctrlX Datalayer .NET API.

**IMPORTANT**: If this method is called, the subscription is unusable and subsequent calls to any Subscribe method return with DL_INVALID_ADDRESS.
